increment.js

```diff
@@ -0,0 +1,50 @@
+const fs = require('fs-extra');
+const os = require('os');
+const path = require('path');
+
+const config = require('./config')
+
+const issueCommentsJson = `${config.source.repo}/issue-comments.json`;
+const prCommentsJson = `${config.source.repo}/pull-comments.json`;
+
+const issueComments = JSON.parse(fs.readFileSync(issueCommentsJson));
```

Comment:
This files uses a lot of `sync` versions of the `fs` library. This is fine. Other scripts use the async version with the `await` keyword mostly to allow for any async call (including network calls). The caveat is having to have an async function that gets called (like `main`).

If it only uses the file system, it is probably fine.